011-quarkus-panache-rest-flyway/src/main/java/io/quarkus/qe/ApplicationEntity.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.qe;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.validation.constraints.NotEmpty;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+
+@Entity(name = "application")
+public class ApplicationEntity extends PanacheEntity {
+    @NotEmpty
```

Comment:
The @NotEmpty annotation is not working. I reported this issue to make the extensions worked together: 
https://github.com/quarkusio/quarkus/issues/13307

Reply:
The discussion on the issue is not indicating that `@NotEmpty` is not working. Can you please update the issue with more details? Or update the disabled test.

Reply:
I've updated the disabled test to provide more details about why is failing. 
Let me know if it looks ok now.
Thanks.